app/Http/Controllers/PostsController.php

```diff
@@ -145,7 +145,7 @@ public function update(Request $request, Post $post)
         ]);
 
         // Only allow an admin/staff or the user who owns the post to update.
-        if (in_array(token()->role, ['admin', 'staff']) || auth()->id() === $post->northstar_id) {
+        if (is_staff_user() || auth()->id() === $post->northstar_id) {
```

Comment:
At some point, I'd love to extract these into [policy classes](https://laravel.com/docs/5.6/authorization#creating-policies)!

Reply:
oohh yes @sbsmith86 what you were talking about [here](https://github.com/DoSomething/rogue/pull/667#discussion_r180554588)!

Reply:
[Here](https://www.pivotaltracker.com/n/projects/2019429/stories/156700437) is the card for this :)